publication-commons/src/main/java/no/unit/nva/publication/model/storage/FileDao.java

```diff
@@ -65,24 +95,45 @@ public String indexingType() {
         return TYPE;
     }
 
+    @Override
+    @JsonProperty(BY_TYPE_CUSTOMER_STATUS_INDEX_PARTITION_KEY_NAME)
+    public String getByTypeCustomerStatusPartitionKey() {
+        return null;
+    }
+
+    @Override
+    @JsonProperty(BY_TYPE_CUSTOMER_STATUS_INDEX_SORT_KEY_NAME)
+    public String getByTypeCustomerStatusSortKey() {
+        return null;
+    }
```

Comment:
This is important so we do not persist database entries in unnecessary index, for example as we do with message. 
Something like Customer:315981-12513-125-124124:Status:NOT_IMPLEMENTED. 

This implementation will not persist any entry in ByTypeCustomerStatus secondary index.

Reply:
For once, I'd add a comment to this effect